StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -25,6 +25,8 @@ extension PaymentSheet {
                 return [.returnURL]
             case .dynamic("zip"):
                 return [.returnURL]
+            case .dynamic("alipay"):
```

Comment:
We already defined this on Line 26.  I think we should try to avoid `.dynamic` if the type already a STPPaymentMethodType

Reply:
Oh, I didn't notice this was only for PMs not represented by STPPaymentMethodType, is not clear unless you follow the call